fall18/p2/README.md

```diff
@@ -1,3 +1,149 @@
 # Project 2
 
-More details coming soon.
+In this project, you'll learn about creating and calling functions, arguments, return values and math module. To start, download these files:
```

Comment:
Let's add a note saying to right click the links below, not the links above that only show a preview